### DISCUSSION TITLE
about mysql revision

### DISCUSSION THREAD

**Discussion by zhongminke** [blueseerERP/blueseer#47] [Q&A] [answered]
Hello
I want to know server preparation for mysql revision. For example:
1. need create bsdb database on mysql server?
2. about "**Enter the administrator password for the MySQL Database:**",administrator  means "root" or "bs_user" or "admin" or other？
3. The password MUST be bspasswd or not?
4. if I want to rewrite bs.cfg, how can I do？

thanks,

**Comment by vaughnte** [comment, marked as answer]
The MySQL installation is completely separate from the blueseer installation.   MySQL must be installed 'before' you install blueseer if you downloaded the blueseer.mysql.win.v61 package.    When you install MySQL, you will typically set a mysql administrator password.   This is the password the script will require.  Also, you do not have to create database bsdb in mysql prior.   The blueseer installation script will create it.    Visit the MySQL website and review the installation documentation.

**Reply by zhongminke** [reply]
thanks,

means I need create a user on mysql with name “administrator ”，right？

**Reply by vaughnte** [reply]
MySQL installation by default has a user 'administrator'...you just need to set the password.   See the mysql documentation on their website.

**Comment by zhongminke** [comment]
sorry, there are still some ERRORs ,can you help me?

![image](https://user-images.githubusercontent.com/20702367/160321135-305681c8-9af7-4d1d-8357-921f6161ba95.png)

**Reply by vaughnte** [reply]
I just tried the script (on mysql version 8), and I do not get the same errors...so there may be a variation in your mysql installation.    The my.cnf file created is a temporary file with your credentials you entered.   You might try commenting out the section in the install script where it deletes the my.cnf after installation...... and view it in a text editor.   Check for bad characters.   It should look like this:  

[client]
user = root
password = xxx
host = localhost
local_infile = ON

...where xxx is your root password you set when you installed mysql.

You might want to look at this post as well:
https://stackoverflow.com/questions/8020297/mysql-my-cnf-file-found-option-without-preceding-group
